HHS/Head-Start-TTADP
# [TTAHUB-795] Save AR Collaborator Role

## Description of change

When a collaborator is added to an AR we want to save their roles "at that time" with the report.

## How to test

- Verify landing page correctly shows Alerts and Approved report Collaborators. 
- Create / Update / Reset / Submit / Approve Report
- Should test a case where a collaborator is added without roles. Then the same report is saved when the user has roles added.
- On the AR landing page the Alerts table should show collaborators saved on reports (and export).
- On the AR landing page the Approved AR table should show what's saved on the report else user roles (and export).

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-795


## Checklists

### Every PR


~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy


### After merge/deploy
